internal/terraform/context_apply.go

```diff
@@ -38,6 +38,17 @@ func (c *Context) Apply(plan *plans.Plan, config *configs.Config) (*states.State
 		return nil, diags
 	}
 
+	for _, rc := range plan.Changes.Resources {
+		// import is a no-op change, but we'd like to show some helpful output that mirrors
+		// the way we show other changes.
+		if rc.Importing != nil {
+			for _, h := range c.hooks {
+				h.PreApplyImport(rc.Addr, *rc.Importing)
+				h.PostApplyImport(rc.Addr, *rc.Importing)
```

Comment:
It feels a bit silly to call both hooks here, but I couldn't find a better place to call `PostApplyImport` 😅 should we keep the two separate pre/post hooks anyway for forward-thinking purposes?

Reply:
I'm okay with keeping two to ensure that it's future proofed, maybe just add a comment explaining why we we're calling both immediately?

Reply:
Makes sense to me!